<?xml version="1.0" encoding="UTF-8"?><testrun name="HibernateUtilTeste" project="TrainingWebService" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="br.com.farmacia.util.HibernateUtilTeste" time="1.099">
    <testcase name="conectarHibernate" classname="br.com.farmacia.util.HibernateUtilTeste" time="1.099">
      <error>java.lang.ExceptionInInitializerError&#13;
	at br.com.training.util.HibernateUtil.criarSessionFactory(HibernateUtil.java:42)&#13;
	at br.com.training.util.HibernateUtil.&lt;clinit&gt;(HibernateUtil.java:15)&#13;
	at br.com.farmacia.util.HibernateUtilTeste.conectarHibernate(HibernateUtilTeste.java:12)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: org.hibernate.AnnotationException: No identifier specified for entity: br.com.training.model.Equipamento&#13;
	at org.hibernate.cfg.InheritanceState.determineDefaultAccessType(InheritanceState.java:277)&#13;
	at org.hibernate.cfg.InheritanceState.getElementsToProcess(InheritanceState.java:224)&#13;
	at org.hibernate.cfg.AnnotationBinder.bindClass(AnnotationBinder.java:775)&#13;
	at org.hibernate.cfg.Configuration$MetadataSourceQueue.processAnnotatedClassesQueue(Configuration.java:3845)&#13;
	at org.hibernate.cfg.Configuration$MetadataSourceQueue.processMetadata(Configuration.java:3799)&#13;
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1412)&#13;
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1846)&#13;
	at br.com.training.util.HibernateUtil.criarSessionFactory(HibernateUtil.java:38)&#13;
	... 25 more&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
